packages/hdwallet-xdefi/src/xdefi.ts

```diff
@@ -93,6 +93,7 @@ export class XDEFIHDWallet implements core.HDWallet, core.ETHWallet {
   readonly _supportsOptimism = true;
   readonly _supportsBSC = true;
   readonly _supportsPolygon = true;
+  readonly _supportsGnosis = true;
```

Comment:
also not listed as supported
```suggestion
  readonly _supportsGnosis = false;
```
![image](https://github.com/shapeshift/hdwallet/assets/35275952/48b83246-04db-47ba-a84c-1dea4047cc4c)